Common/Data/Collections/FastVec.h

```diff
@@ -155,3 +155,16 @@ class FastVec {
 	bool capacityLocked_ = false;
 #endif
 };
+
+// Simple cyclical vector.
+// Initially, doesn't do any sanity checking.
+template <class T, size_t size>
+class HistoryBuffer {
+public:
+	// Out of bounds (past size() - 1) is undefined behavior.
```

Comment:
Isn't it explicitly not?  Isn't that the point of this class?

-[Unknown]

Reply:
Well the comment is wrong since there's no size(), only the constant looping buffer `size` which is a different concept .. Originally I intended to track the highest value here (as size()) and reject accesses that were `size` below it. Of course that naming is stupid and would hae been changed, but I'm sure you get the idea - in an always growing buffer like this where you only care about the last 32 events say, accessing events 33 below the top is bad.